src/Sulu/Bundle/SecurityBundle/Admin/SuluSecurityContactsContentNavigationProvider.php

```diff
@@ -2,30 +2,36 @@
 
 namespace Sulu\Bundle\SecurityBundle\Admin;
 
-use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationInterface;
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationProviderInterface;
 use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationItem;
 use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
 
-class SuluSecurityContactsContentNavigation implements ContentNavigationInterface
+class SuluSecurityContactsContentNavigationProvider implements ContentNavigationProviderInterface
```

Comment:
Don't really know how to handle this... I also already thought about removing the `Sulu`-prefix for this and the admin classes, would also make the class name a bit shorter. Does anyone see a value in these prefixes? /cc @chirimoya @dantleech @wachterjohannes